settlement/grpc/grpc.go

```diff
@@ -228,6 +228,11 @@ func (c *Client) GetProposer() *types.Sequencer {
 	return types.NewSequencer(tmPubKey, pubKey.Address().String())
 }
 
+// GetSequencerByAddress returns all sequencer information by its address. Not implemented since it will not be used in grpc SL
+func (c *Client) GetSequencerByAddress(address string) (types.Sequencer, error) {
+	return types.Sequencer{}, nil
```

Comment:
better to return error with empty sequencer?

Reply:
added panic because it shouldnt be called from grpc sl